community/modules/compute/gke-job-template/templates/gke-job-base.yaml.tftpl

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ${name}${suffix}
+spec:
+  parallelism: ${node_count}
+  completions: ${node_count}
+  template:
+    spec:
+      nodeSelector:
+      %{~ if machine_family != null ~}
+        cloud.google.com/machine-family: ${machine_family}
+      %{~ endif ~}
+      %{~ if node_pool_name != null ~}
```

Comment:
While node pool is optional, can it be omitted at job submission? Can this be more than a pool? (submitting to multiple available pools?)

Reply:
Both options are supported: no node pool, multiple node pools.